2018/python/file_generator.py

```diff
@@ -468,10 +468,15 @@ def edit_report(self):
 				getattr(checker, func)()
 
 		#Creates a dataframe of results from the checker. 
-		res_df = pd.DataFrame(checker.results)
+		report_df = pd.DataFrame(checker.results)
 
-		#Filters the results for edits that have failed. 
-		res_df = res_df[(res_df['status']=='failed')]
+		#Creating a report based on the status of the edits failed.
+		if self.edit_report_config['status'] == 'failed':
```

Comment:
Is there a reason to not combine all the edit results into a single report?

It seems like having a single report would simplify things and one could use the status in the report to filter for needed results.

Reply:
Sure - I think that would be fine to output a full edit report.